### ISSUE TITLE
Traverse error on 0.9.1

### ISSUE THREAD

**Issue by KristofferC** [JuliaEditorSupport/LanguageServer.jl#250]
Backtrace:

```
ERROR: LoadError: TypeError: Expr: expected Symbol, got CSTParser.LITERAL
Stacktrace:
 [1] Expr(::Any) at ./boot.jl:192
 [2] toplevel(::CSTParser.EXPR{CSTParser.Block}, ::LanguageServer.TopLevelScope, ::LanguageServer.LanguageServerInstance) at /Users/kristoffer/.vscode/extensions/julialang.language-julia-0.9.1/scripts/languageserver/packages/LanguageServer/src/trav/toplevel.jl:45
 [3] toplevel(::CSTParser.EXPR{CSTParser.If}, ::LanguageServer.TopLevelScope, ::LanguageServer.LanguageServerInstance) at /Users/kristoffer/.vscode/extensions/julialang.language-julia-0.9.1/scripts/languageserver/packages/LanguageServer/src/trav/toplevel.jl:43 (repeats 3 times)
 [4] toplevel(::CSTParser.EXPR{CSTParser.FileH}, ::LanguageServer.TopLevelScope, ::LanguageServer.LanguageServerInstance) at /Users/kristoffer/.vscode/extensions/julialang.language-julia-0.9.1/scripts/languageserver/packages/LanguageServer/src/trav/toplevel.jl:40
 [5] lint(::LanguageServer.Document, ::LanguageServer.LanguageServerInstance) at /Users/kristoffer/.vscode/extensions/julialang.language-julia-0.9.1/scripts/languageserver/packages/LanguageServer/src/trav/lint.jl:41
 [6] process(::LanguageServer.JSONRPC.Request{Val{Symbol("workspace/didChangeConfiguration")},Dict{String,Any}}, ::LanguageServer.LanguageServerInstance) at /Users/kristoffer/.vscode/extensions/julialang.language-julia-0.9.1/scripts/languageserver/packages/LanguageServer/src/provider_misc.jl:242
 [7] run(::LanguageServer.LanguageServerInstance) at /Users/kristoffer/.vscode/extensions/julialang.language-julia-0.9.1/scripts/languageserver/packages/LanguageServer/src/languageserverinstance.jl:38
 [8] include_from_node1(::String) at ./loading.jl:576
 [9] include(::String) at ./sysimg.jl:14
 [10] process_options(::Base.JLOptions) at ./client.jl:305
 [11] _start() at ./client.jl:371
while loading /Users/kristoffer/.vscode/extensions/julialang.language-julia-0.9.1/scripts/languageserver/main.jl, in expression starting on line 27
```

**Comment by ZacLN** [comment]
Could you show me the relevant code? It should be a call to `include` in the file

**Comment by KristofferC** [comment]
It's in the FemtoCleaner repo: https://github.com/JuliaComputing/FemtoCleaner.jl

It prints

```
INFO: Set(String["/Users/kristoffer/.julia/v0.6/FemtoCleaner"])
INFO: parsed /Users/kristoffer/.julia/v0.6/FemtoCleaner/bin/statistics.jl
INFO: parsed /Users/kristoffer/.julia/v0.6/FemtoCleaner/src/FemtoCleaner.jl
INFO: parsed /Users/kristoffer/.julia/v0.6/FemtoCleaner/src/autodeployment.jl
INFO: parsed /Users/kristoffer/.julia/v0.6/FemtoCleaner/src/blame.jl
INFO: parsed /Users/kristoffer/.julia/v0.6/FemtoCleaner/src/interactions.jl
INFO: parsed /Users/kristoffer/.julia/v0.6/FemtoCleaner/test/dry_runs.jl
INFO: parsed /Users/kristoffer/.julia/v0.6/FemtoCleaner/test/runtests.jl
ERROR: LoadError: TypeError: Expr: expected Symbol, got CSTParser.LITERAL
Stacktrace:
 [1] Expr(::Any) at ./boot.jl:192
 [2] toplevel(::CSTParser.EXPR{CSTParser.Block}, ::LanguageServer.TopLevelScope, ::LanguageServer.LanguageServerInstance) at /Users/kristoffer/.vscode/extensions/julialang.language-julia-0.9.1/scripts/languageserver/packages/LanguageServer/src/trav/topl.
.
.
```

**Comment by davidanthoff** [comment]
@ZacLN do you want to fix this for v0.10, or move to the next release?

**Comment by davidanthoff** [comment]
I'm moving this to the next release.
